velox/common/dynamic_registry/tests/DynamicLinkTest.cpp

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Its a bit difficult to go between the tests and the function files they are using. Might be good to name them similarly... like if the function name is dynamic_1 then the filename could be MyDynamic1...